src/main/java/com/notify/notify/model/response/CodeResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.notify.notify.model.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CodeResponse {
```

Comment:
please do read read about this annotation  @JsonInclude(JsonInclude.Include.NON_NULL), and see if we can use here.

Reply:
I have read it before. i think, it doesn't make sense to add it here.